src/components/Banner/Banner.scss

```diff
@@ -113,6 +113,7 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
 }
 
 .Banner {
+  --p-rgb-text: 33, 43, 54;
```

Comment:
Need to add the custom property here for non new design language views. For now I've left it scoped to `.Banner`